src/main/kotlin/com/tripl3dogdare/havenjson/Utils.kt

```diff
@@ -38,3 +36,8 @@ internal val <T, C : Collection<T>> C.head get():T? = elementAtOrNull(0)
 internal val <T, C : Collection<T>> C.tail get():List<T> = drop(1)
 internal operator fun Regex.contains(text:Char):Boolean = contains(text.toString())
 internal operator fun Regex.contains(text:CharSequence):Boolean = this.matches(text)
+
+private fun String.replaceAll(mapping: Map<String, String>) =
```

Comment:
Do you think you could make this `internal` instead of `private`, similar to the other methods in this file? That way it can potentially be used elsewhere in the codebase while still not leaking to the public API.